modules/shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/spark/ShredJob.scala

```diff
@@ -62,6 +67,10 @@ object ShredJob extends SparkJob {
 
   val processor = Processor(ProjectMetadata.name, ProjectMetadata.version)
 
+  final val SQS_MAX_RETRIES = 10
```

Comment:
We generally try follow Scala Style Guide with constants naming: https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods